survey_multiple_choice_max_answer/models/survey_question.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SurveyQuestion(models.Model):
+
+    _inherit = "survey.question"
+
+    validation_multiple_answers_min = fields.Integer(
+        "Minimum Number of Answers", default=0
```

Comment:
The default value of an integer is 0. You don't have to specify it

Reply:
Right, changed